openhantek/src/main.cpp

```diff
@@ -92,15 +92,15 @@ int main(int argc, char *argv[]) {
 #endif
 
     bool useGles = false;
-    {
+    if(useGles) {
```

Comment:
That looks absolutely wrong. It is dead code now

Reply:
Agree, that's ugly. But it was more wrong :)
This part between curly brackets all the time returned true
``` c++
useGles = p.isSet(useGlesOption);
```
and without if in any case I felt into this part:
``` c++
bool useGles = false;
{
....
}
```
and as result youend up with useGles=true and fall into openGL mode

Reply:
It is not just ugly it disables the command line parsing completly. The behaviour was submitted by a user with the argument that the command line argument should always overwrite the auto detection. And yes OpenGL es ist the preferred render engine if both are detected.

Reply:
oh yes I've  sorry I've did stupid thing because usually used getopt
and misunderstood qt doc ) 
actually it works like it should. But yes that' what I thought that auto detection should be there )
So yes you are right I've did stupid thing here